src/configurations/destinations/webhook/schema.json

```diff
@@ -517,7 +517,7 @@
       },
       "webhookUrl": {
         "type": "string",
```

Comment:
@Gauravudia there's one more recommendation w.r.t IPV6 address too that's not handled